src/@carbon-info/views/Data/Data.tsx

```diff
@@ -1,217 +1,243 @@
-        });
-      }
-      setTableInfo(results);
-    }
-    fetchDataItems();
-  }, [data]);
+			if (content && Array.isArray(content.items)) {
+				content.items.forEach((o: any) => {
+					let data: DataInfo = {
+						value: o.fields.value,
+						description: o.fields.key,
+						toCountUp:
+							o.fields.key === "On-Chain_Transactions" ||
+							o.fields.key === "Total_Value_Locked" ||
+							o.fields.key === "Total_SWTH_Staked",
+					};
+					results.push(data);
+				});
+			}
+			setTableInfo(results);
+		}
+		fetchDataItems();
+	}, [data]);
 
-  return (
-    <div ref={ref} id="data" className={classes.container}>
-      <FadeAndSlide visible={inView}>
-        <div className={classes.poweredBy}>
-          <Typography variant="h2" color="textPrimary" className={clsx(classes.headerText, classes.smallerText, { open: inView })}>
-            Powered by
-          </Typography>
-          <img src={cosmosSDK} alt="Cosmos SDK" className={classes.cosmosSDKLogo} />
-          <Typography variant="h2" color="textPrimary" className={clsx(classes.headerText, classes.cosmosSDK, classes.smallerText, { open: inView })}>
-            Cosmos SDK
-          </Typography>
-        </div>
-        <Typography variant="h2" color="textPrimary" className={clsx(classes.headerText, { open: inView })}>
-          Carbon allows anyone to bootstrap
-          <br />
-          open financial markets for
-          <br />
-          any asset type on any blockchain.
-        </Typography>
-      </FadeAndSlide>
-      <FadeAndSlide visible={inView} className={classes.dataWrapper}>
-        <AnimateKeyframes
-          play={inView}
-          iterationCount={1}
-          keyframes={["transform: scale(0)", "transform: scale(1)"]}
-          duration={0.75}
-        >
-          <Grid container item xs={8} sm={12} spacing={0} justifyContent="center" className={clsx(classes.dataTable, { open: inView })}>
-            {tableInfo.map((item: DataInfo) => (
-              <Grid item xs={12} sm={4} xl={2} key={item.description}>
-                <Paper className={classes.dataBox} elevation={0}>
-                  {item.toCountUp ? <RollingNum item={item} /> : <Typography variant="h3" color="textPrimary" align="center">{item.value}</Typography>}
-                  <Typography variant="body2" color="textSecondary" align="center">{item.description.replaceAll("_", " ")}</Typography>
-                </Paper>
-                {widthXs && <Divider className={classes.mobileDivider} />}
-              </Grid>
-            ))}
-          </Grid>
-        </AnimateKeyframes>
-      </FadeAndSlide>
-    </div >
-  );
+	return (
+		<div ref={ref} id="data" className={classes.container}>
+			<FadeAndSlide visible={inView} className={classes.dataWrapper}>
+				<AnimateKeyframes
+					play={inView}
+					iterationCount={1}
+					keyframes={["transform: scale(0)", "transform: scale(1)"]}
+					duration={0.75}
+				>
+					<Grid
+						container
+						item
```

Comment:
Is it possible for a Grid component to be both `container` and `item` at the same time?
If not, do remove 1 of them.